src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -786,7 +786,7 @@ namespace HVACVariableRefrigerantFlow {
 			TotalTUHeatingCapacity = TotalCondHeatingCapacity * VRF( VRFCond ).PipingCorrectionHeating;
 			if ( TotalCondHeatingCapacity > 0.0 ) {
 				HeatingPLR = ( TUHeatingLoad / VRF( VRFCond ).PipingCorrectionHeating ) / TotalCondHeatingCapacity;
-				HeatingPLR += LoadDueToDefrost / TotalCondHeatingCapacity;
+				HeatingPLR += ( LoadDueToDefrost * HeatingPLR ) / TotalCondHeatingCapacity;
 			} else {
```

Comment:
I don't think I can iterate on this. TUHeatingLoad, PipingCorrectionHeating, TotalCondHeatingCapacity, and LoadDueToDefrost are all constants.